plugins/arDominionPlugin/css/less/bootstrap-mod.less

```diff
@@ -27,3 +27,8 @@
 .open > .dropdown-menu {
   z-index: 10004
 }
+
+// Upper padding for alerts
+.alert {
+  margin-top: 1em
```

Comment:
One issue with this approach is that it's affecting all the cases where `.alert` is used, see `git grep -E "class=\".*alert.*\""`. Should you target #notice-alerts instead? Not entirely sure but is it worth trying to find a better id as well? e.g. `#app-alert` or `#global-alert` or similar?